app/api/events.py

```diff
@@ -111,6 +112,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    speaker = Relationship(attribute='speaker',
```

Comment:
A single event can have multiple speakers right ? So why is this called as `speaker` instead of `speakers` ?